### ISSUE TITLE
We should hide private methods

### ISSUE THREAD

**Issue by birtles** [web-animations/web-animations-js-legacy#514]
Some users are clearly calling these methods that start with underscores. That's less than ideal since their code will break when native implementations are available. It also means we're not getting feedback about the API as-specced.

**Comment by dstoc** [comment]
That's unfortunate. The underscore should be a pretty good hint that these methods are not intended to be called.

A couple of options:
- Pass tokens as we do now to prevent certain constructors from being called.
- Pull the methods off the prototypes, and .call them?
- Promote usage of a minified build where symbols other than what we choose to export are obfuscated.

**Comment by birtles** [comment]
Yeah, we could use private functions but then they don't end up on the prototype chain and I guess we need that (e.g. this sort of approach: http://stackoverflow.com/questions/55611/javascript-private-methods). Maybe it's not worth the effort of rewriting things to work around this.

**Comment by mithro** [comment]
Do you have examples of people calling them? Maybe they have a use case we haven't covered?

**Comment by birtles** [comment]
There is this sort if thing:
http://lists.w3.org/Archives/Public/public-fx/2013OctDec/0096.html

```
It seems to be mostly just not understanding that '_' means 'don't use'. 
So in this message the author is using '_player' instead of 'player'.
```

**Comment by mithro** [comment]
I think that user knows they are using the private API, they say

```
I would be forced to use private API to clean up animations not contained in a group:
anim.onend = function() { 
    if (anim.parent) anim.remove();
    else anim._player._deregisterFromTimeline();
}
```

**Comment by birtles** [comment]
That's only because I pointed it out in a separate thread. (And still he keeps on using it.)
